### DISCUSSION TITLE
Could not find a declaration file for module '@observablehq/plot'

### DISCUSSION THREAD

**Discussion by sswatson** [observablehq/plot#1366] [Q&A] [answered]
If I start a new Typescript project in Node in Node, then `npm install @observablehq/plot`, and do `import * as Plot from '@observablehq/plot`, then I get the following Typescript error:
```
Could not find a declaration file for module '@observablehq/plot'
```
I feel like I'm making some silly Typescript mistake, but I can't figure out what it is. Sort of seems like maybe other folks will have encountered this, so I thought I'd put it out there.

**Comment by mbostock** [comment, marked as answer]
We do not yet publish TypeScript declarations. They were added in #1320 (see #401) and will be available in the next release of Plot. We are currently adding documentation before releasing. #1343
